source/administration/mmctl-cli-tool.rst

```diff
@@ -2990,6 +2991,42 @@ LDAP-related utilities.
 
    -h, --help   help for ldap
 
+mmctl ldap idmigrate
+~~~~~~~~~~~~~~~~~~~~
+
+**Description**
+
+Migrate LDAP ``IdAttribute`` to a new value. Run this utility then change the IdAttribute to the new value.
+
+**Format**
+
+.. code-block:: sh
+
+   mmctl ldap idmigrate [flags]
```

Comment:
The usage should include the `objectGUID` as a positional argument. This should be added to the `mmctl` code as well.

```suggestion
   mmctl ldap idmigrate <objectGUID> [flags]
```

Reply:
I've just sent a PR with this change to `mmctl` here: https://github.com/mattermost/mmctl/pull/356